package.json

```diff
@@ -4,22 +4,25 @@
   "description": "A connection (pool) to PostgreSQL available anywhere in your hapi application",
   "main": "index.js",
   "scripts": {
-    "start": "PORT=8000 node test/server_example.js",
-    "create": "node test/_create_test_db.js",
-    "fast": "./node_modules/tape/bin/tape ./test/*.js",
-    "perf": "node test/perf.test.js",
-    "test": "npm run create && ./node_modules/.bin/istanbul cover ./node_modules/tape/bin/tape ./test/connection_error.test.js  | node_modules/tap-spec/bin/cmd.js && npm run perf"
+    "start": "PORT=8000 node ./tests/server/server_example.js",
+    "createTestDB": "node ./tests/_create_test_db.js",
+    "allTests": "./node_modules/tape/bin/tape ./tests/*.js",
+    "performanceTest": "PORT=8000 nyc node ./tests/perf.test.js | tap-spec",
+    "connectionErrorTest": "PORT=8000 nyc node ./tests/connection_error.test.js | tap-spec",
```

Comment:
note sure if this would help, can you try adding the flag `--reporter=lcov` to `nyc`

Reply:
think it's works